examples/program/BUILD.bazel

```diff
@@ -12,15 +13,20 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+mock_typescript_lib(
+    name = "tots_typescript",
```

Comment:
cute name, but let's just be descriptive, like "mock_ts_lib" orsomething

Reply:
Done.